src/pages/aclmanagement.html

```diff
@@ -14,155 +14,552 @@ <h1 id="acl-management-view-title">
-                                      style-class="form-control form-control-sm"
-                                      namespaces="namespaces"
-                                      autocomplete-status-loader="getAutocompletePromise"
-                                      placeholder="{{'acl_management.rulestable.field_placeholders.object' | translate}}"></autocomplete>
-                        <em></em>
-                    </td>
-                    <td class="data context-cell">
-                        <autocomplete ng-model="rule.context" name="context" required
-                                      style-class="form-control form-control-sm"
-                                      namespaces="namespaces"
-                                      autocomplete-status-loader="getAutocompletePromise"
-                                      placeholder="{{'acl_management.rulestable.field_placeholders.context' | translate}}"></autocomplete>
-                        <em></em>
-                    </td>
-                    <td class="data role-cell">
-                        <input type="text" name="role" required ng-model="rule.role" uppercased
-                               autocomplete="off" class="form-control form-control-sm"
-                               placeholder="{{'acl_management.rulestable.field_placeholders.role' | translate}}"/>
-                        <em></em>
-                    </td>
-                    <td class="data policy-cell">
-                        <select ng-model="rule.policy" class="form-control form-control-sm">
-                            <option selected>allow</option>
-                            <option>deny</option>
-                        </select>
-                    </td>
-                    <td class="actions-cell">
-                        <button ng-click="cancelEditing($index)" class="btn btn-link cancel-rule-editing-btn"
-                                gdb-tooltip="{{'acl_management.rulestable.actions.cancel_rule_editing' | translate}}">
-                            <em class="icon-close"></em>
-                        </button>
-                        <button ng-if="ruleData.$valid" ng-click="saveRule()"
-                                class="btn btn-link save-rule-btn"
-                                ng-disabled="!ruleData.$valid"
-                                gdb-tooltip="{{'acl_management.rulestable.actions.save_rule' | translate}}">
-                            <em class="icon-save"></em>
-                        </button>
-                        <em ng-if="!ruleData.$valid" class="icon-save icon-lg save-rule-disabled-btn"
-                            gdb-tooltip="{{'acl_management.rulestable.messages.invalid_form' | translate}}"></em>
-                    </td>
-                </tr>
-                <tr ng-repeat-end ng-if="0"></tr>
-                </tbody>
-            </table>
-        </form>
-
+        <div class="acl-management-container">
```

Comment:
I wonder if it isn't about time to split this to smaller components/directives/

Reply:
I was thinking of making the form dynamic based on the different scopes and their properties, but that would take time. We may plan it in the next iterations